src/lib.rs

```diff
@@ -84,3 +106,44 @@ fn makecircle(centerpoint: Point<f64>, radius: f64, num_vertices: usize) -> Poly
     let polygon = Polygon::new(LineString::from(circle_points), vec![]);
     polygon
 }
+
+// Make a single clock polygon
+pub fn clockpoly(
+    centerpoint: Point<f64>,
+    radius: f64,
+    inner_radius: f64,
+    num_vertices: usize,
+    num_segments: usize,
+    seg: usize,
+) -> Polygon<f64> {
+    let mut circle_points = Vec::new();
+    let mut circle_points_inner = Vec::new();
+
+    // Sequence of vertices
+    // in R round(seq(1, 13, length.out = 12))
+    // Number of vertices per segment
+    let n = num_vertices / num_segments;
+    let f = 0 + (seg) * n;
+    let t = 0 + (seg + 1) * n;
+    // Outer radius
```

Comment:
I think another helper function would make this much more clear:

`fn arc(angle1: f64, angle2: f64, num_vertices: usize, radius: f64, center: Point<f64>) -> Vec<Point<f64>>`